tests/unit/nupic/research/temporal_memory_test.py

```diff
@@ -210,36 +210,6 @@ def testReinforceCorrectlyActiveSegments(self):
     self.assertAlmostEqual(.42, tm.connections.dataForSynapse(is1).permanence)
 
 
-  def testNoGrowthOnCorrectlyActiveSegments(self):
```

Comment:
It stopped being valid with the recent algorithm change https://github.com/numenta/nupic/pull/3293

It was explicitly checking that we don't grow synapses on predicted columns. Now we do.

The test was flawed (my bad) -- it should be failing now, but instead it only sometimes fails, depending on the random number generator. It didn't do a good job of controlling the winner cells, so sometimes no synapse growth happens in today's code. If I were writing this test today, I'd use 1 cell per column to control the winner cells.